server/app/services/export/PdfExporter.java

```diff
@@ -19,9 +20,12 @@
 import java.time.ZoneId;
 import javax.inject.Inject;
 import models.Application;
+import services.Path;
 import services.applicant.AnswerData;
 import services.applicant.ApplicantService;
 import services.applicant.ReadOnlyApplicantProgramService;
+import services.applicant.question.Scalar;
+import services.question.types.QuestionType;
 
 /** PdfExporter is meant to generate PDF files. The functionality is not fully implemented yet. */
```

Comment:
Mind removing the "is not fully implemented yet" part of the docstring? If there's still functionality left to be implemented, I'd consider explicitly listing it out (with a tracking bug).

Reply:
Good catch! Removed it.  Everything that was requested is implemented.